### PR TITLE
Create NCAR_Schuster.yaml

### PR DESCRIPTION
**Pull request by showmic09**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Hi @opensciencegrid/project-office , can we get approval for the new organization "National Center for Atmospheric Research" ?

**Comment by jstathas** [comment]
Approved.

I do have a question as to process – who is adding these to the institution database once approved?

For example the last one I approved, when I went to add it a couple of hours later it was already there. I want a designated responsible party – I’m not clear on the backend if there is something shoving the new values into the DB.

Janet

From: Matyas Selmeci ***@***.***>
Date: Thursday, May 2, 2024 at 3:25 PM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Team mention ***@***.***>
Subject: Re: [opensciencegrid/topology] Create NCAR_Schuster.yaml (PR #3855)

Hi @opensciencegrid/project-office<https://github.com/orgs/opensciencegrid/teams/project-office> , can we get approval for the new organization "National Center for Atmospheric Research" ?

File: projects/NCAR_Schuster.yaml
Comment (matyasselmeci) [inline projects/NCAR_Schuster.yaml:4]:
```suggestion
Description: >-
  We plan to test the capabilities of the OSPool to run climate science
  research related data analytics workflows using datasets made accessible through
  NSF NCAR's integration with the Open Science Data Federation.
```

**Comment by matyasselmeci** [comment]
Sounds like we still need to work out the procedure and inform the staff of what it is. Last time, the .yaml file had an "InstitutionID: unknown"; Matt took care of adding the org to the database so I could fill that value in.

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**
